app/api/resources/mentorship_relation.py

```diff
@@ -210,7 +210,7 @@ class AcceptMentorshipRelation(Resource):
     @mentorship_relation_ns.response(
         HTTPStatus.NOT_FOUND, "%s" % messages.MENTORSHIP_RELATION_REQUEST_DOES_NOT_EXIST
     )
-    def put(cls, request_id):
+    def put(cls, relation_id):
```

Comment:
No need to do any of the changes(i.e from request_id to relation_id) in this function. Since, we still haven't accepted the relation request.